From 7070427158646f7b24f0510ddedece5f3cf65c20 Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Wed, 6 Jul 2016 22:47:19 +0800
Subject: [PATCH 0584/2241] Revert "fix: spi: a3700: disable the fifo mode"

This reverts commit 453805cb52978476c5f82f7e9c31bef4e086a688
("fix: spi: a3700: disable the fifo mode").

Change-Id: I8b62746309014349eded93af18471996a441add2
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30970
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Victor Gu <xigu@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index 026dd68..1825522 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -111,6 +111,7 @@
 				cell-index = <0>;
 				clock-frequency = <160000>;
 				max-frequency = <40000>;
+				fifo-mode;
 				interrupts = <0 0 4>;
 				status = "disabled";
 			};
-- 
2.7.4

